### DISCUSSION TITLE
Updating profile picture fails with authorization error

### DISCUSSION THREAD

**Discussion by felixfbecker** [outline/outline#7729] [Self Hosting]
**To Reproduce**
1. Go to profile settings
2. Upload new picture
4. See authorization error

**Expected behavior**

Should update the profile picture

**Screenshots**

![CleanShot 2024-10-03 at 17 11 17@2x](https://github.com/user-attachments/assets/d5b18055-23ad-4dc6-828d-ede93e62b77d)

**Outline (please complete the following information):**
- Install: self hosted
- Version: not sure

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome
 - Version 129.0.6668.90

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-689/updating-profile-picture-fails-with-authorization-error">OLN-689 Updating profile picture fails with authorization error</a></p>

**Comment by tommoor** [comment]
@felixfbecker fixed in https://github.com/outline/outline/commit/033b05f67928992bee2396c511ba7c1f3403831b – looks like this is only the case when the setting to prevent users from deleting their own account was enabled
